openproblems/data/immune_cells_human_mouse.py

```diff
@@ -0,0 +1,45 @@
+from . import utils
+
+import os
+import scprep
+import tempfile
+
+# sparsified from https://figshare.com/articles/dataset/Benchmarking_atlas-level_data_integration_in_single-cell_genomics_-_integration_task_datasets_Immune_and_pancreas_/12420968/2 # noqa: E501
+URL = "https://figshare.com/ndownloader/files/25717166"
+
+
+@utils.loader(data_url=URL, data_reference="luecken2022benchmarking")
+def load_immune_hm(test=False):
+    """Download immune human data from figshare."""
+    import scanpy as sc
+
+    if test:
+        # load full data first, cached if available
+        adata = load_immune_hm(test=False)
+
+        # Subsample immune data to two batches with 250 cells each
+        adata = adata[:, :500].copy()
+        batch1 = adata[adata.obs.batch == "Oetjen_A"][:250]
+        batch2 = adata[adata.obs.batch == "Freytag"][:250]
+        adata = batch1.concatenate(batch2)
```

Comment:
do we need to watch out for cell type numbers here?

Reply:
also, are these both human batches?

Reply:
yeah this is duplicated from the immune cell human code. do you think it matters for testing? 1 human + 1 mouse batch?

Reply:
no, good point. it doesn't really matter